RFC-0012-lazy-tensor.md

```diff
@@ -22,6 +13,16 @@ integration with XLA is imperfect (based on internal work running some
 models on TPU), it remains the best and most comprehensive
 implementation of lazy tensors to date on PyTorch.
 
+## The Prototype
+Alex has been working on a [refactor of torch_xla](https://github.com/fairinternal/nnc_eager/tree/asuhan/lazy_core_only)
```

Comment:
This should point out to the PyTorch core branch: https://github.com/pytorch/pytorch/tree/lazy_tensor_staging

Reply:
fixed